androidplot-core/src/main/java/com/androidplot/Plot.java

```diff
@@ -433,73 +433,54 @@ private void loadAttrs(AttributeSet attrs, int defStyle) {
 
             Field styleableFieldInR = null;
             TypedArray typedAttrs = null;
-
-            final String appPkg = getContext().getPackageName();
-            Class styleableClass = null;
+            
+            Class styleableClass = R.styleable.class;

```

Comment:
If we still cared (we dont though - see comment below) about .jar support, I believe this line would regress with a NoClassDefFoundError being thrown.